distribution/docs/src/main/resources/_contents/_endpoints/csw-endpoint-contents.adoc

```diff
@@ -0,0 +1,2118 @@
+                                    <northBoundLatitude>
+                                        <gco:Decimal>32.323333</gco:Decimal>
+                                    </northBoundLatitude>
+                                </EX_GeographicBoundingBox>
+                            </geographicElement>
+                        </EX_Extent>
+                    </extent>
+                </MD_DataIdentification>
+            </identificationInfo>
+            <distributionInfo>
+                <MD_Distribution>
+                    <distributor>
+                        <MD_Distributor>
+                            <distributorContact/>
+                            <distributorTransferOptions>
+                                <MD_DigitalTransferOptions>
+                                    <onLine>
+                                        <CI_OnlineResource>
+                                            <linkage>
+                                                <URL>http://example.com</URL>
+                                            </linkage>
+                                        </CI_OnlineResource>
+                                    </onLine>
+                                </MD_DigitalTransferOptions>
+                            </distributorTransferOptions>
+                        </MD_Distributor>
+                    </distributor>
+                </MD_Distribution>
+            </distributionInfo>
+        </MD_Metadata>
+    </csw:SearchResults>
+</csw:GetRecordsResponse>
+----
+
+.`GetRecordById` Operation
+The `GetRecords` operation request retrieves the default representation of catalog records using their identifier.
+This operation presumes that a previous query has been performed in order to obtain the identifiers that may be used with this operation.
+For example, records returned by a `GetRecords` operation may contain references to other records in the catalog that may be retrieved using the `GetRecordById` operation.
+This operation is also a subset of the `GetRecords` operation and is included as a convenient short form for retrieving and linking to records in a catalog.
+
+Clients can also retrieve products from the catalog using the `GetRecordById` operation.
+The client sets the output schema to `http://www.iana.org/assignments/media-types/application/octet-stream` and the output format to `application/octet-stream` within the request.
+The endpoint will do the following: check that only one Id is provided, otherwise an error will occur as multiple products cannot be retrieved.
+If both output format and output schema are set to values mentioned above, the catalog framework will retrieve the resource for that Id.
+The HTTP content type is then set to the resource's MIME type and the data is sent out.
+The endpoint also supports the resumption of partial downloads. This would typically occur at the request of a browser when a download was prematurely terminated.
+
+There are two request types: one for `GET
```

Comment:
I haven't found one. It would be nice to control widows & orphans though.

I think some upstream fixed has cleared this instance up, but I'd hate to have to check for these and fix them manually.